aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/RealAWSCognitoAuthPlugin.kt

```diff
@@ -1986,6 +2113,200 @@ internal class RealAWSCognitoAuthPlugin(
         }
     }
 
+    override fun setUpTOTP(onSuccess: Consumer<TOTPSetupDetails>, onError: Consumer<AuthException>) {
```

Comment:
Is there no state machine for this block?

Reply:
yeah its not required as that will be extra overhead. The api is called directly.